### PR TITLE
Lab 7 changes from review

### PR DESCRIPTION
**Pull request by jzaccone**

### CODE REVIEW

**Comment by jzaccone** [comment]
Still need to test permissions, so that will be a following PR

File: workshop/Lab7/README.md [hunk 1]
```diff
@@ -45,7 +72,9 @@ With the IBM Cloud Kubernetes Service clusters at version 1.16 and later, the Op
 
 ### Create an API Key for your Target Account
 
-We will configure the IBM Cloud Operator to maange resources on your personal IBM Cloud Account. You will be able to create and manage a Cloudant DB lite service that only you will have access to.
+We will configure the IBM Cloud Operator to manage resources on your personal IBM Cloud Account. You will be able to create and manage a Cloudant DB lite service that only you will have access to. 
+
+Note: The account that your Cloudant service will be created on MAY be different than the account where your Kubernetes account, so please keep that in mind. If you are participating in a workshop with the IBM Developer Advocacy team, we do this to avoid creating multiple lite cloudantDB services on the shared account where all the k8s clusters are running (IBM Cloud accounts are limited to 1 lite instance per service)
```
Comment (odrodrig) [inline workshop/Lab7/README.md]: The first sentence, "Note: The account that your Cloudant service will be created on MAY be different than the account where your Kubernetes account, so please keep that in mind." doesn't sound right.

Should be something like, "Note: The account that your Cloudant service will be created on MAY be different than the account where your Kubernetes cluster is located, so please keep that in mind."

File: workshop/Lab7/README.md [hunk 2]
```diff
@@ -161,7 +186,7 @@ For an application running within a Kubernetes cluster to be able to access an I
 1. Change into the `yaml` directory. apply the `cloudant-ibmcloud.yaml` file.
 
     ```console
-    cd $HOME/kube-storage101/src/yaml
+    cd $HOME/guestbook-nodejs-config/src/yaml
```
Comment (odrodrig) [inline workshop/Lab7/README.md]: This path does not exist. should be something like "$HOME/guestbook-nodejs-config/storage/lab7"

File: workshop/Lab7/README.md [hunk 3]
```diff
@@ -88,13 +117,19 @@ We will configure the IBM Cloud Operator to maange resources on your personal IB
     ibmcloud iam service-policy-create serviceid-ico --roles Manager --resource-group-name default --region us-south
```
Comment (odrodrig) [inline workshop/Lab7/README.md]: In testing, it turns out you still need the Administrator role added to this permission as well. This line should be, `ibmcloud iam service-policy-create serviceid-ico --roles Manager,Administrator --resource-group-name default --region us-south`

**Comment by jzaccone** [comment]
Admin admin admin. Ok I’ll add that tomorrow

**Comment by jzaccone** [comment]
@odrodrig should be good now

**Comment by jzaccone** [comment]
@odrodrig satisfy review requirement?

**Review by odrodrig — APPROVED** [review]
lgtm

**Pull request merged**